### ISSUE TITLE
E2E tests are falling

### ISSUE THREAD

**Issue by Tal-or** [kubernetes-sigs/node-feature-discovery#962]
**What happened**:
`Node Feature Discovery` e2e tests are falling :
The falling tests are under `test/e2e/node_feature_discovery.go`
1.
 ```
  Nov 21 12:01:01.639: Unexpected error:
      <*errors.errorString | 0xc00147a710>: {
          s: "node \"kind-control-plane\" labels do not match expected, diff (expected vs. received): \u00a0\u00a0map[string]string{\n-\u00a0\t\"feature.node.kubernetes.io/e2e-attribute-test-1\": \"true\",\n-\u00a0\t\"feature.node.kubernetes.io/e2e-flag-test-1\":      \"true\",\n-\u00a0\t\"feature.node.kubernetes.io/e2e-instance-test-1\":  \"true\",\n\u00a0\u00a0}\n",
      }
      node "kind-control-plane" labels do not match expected, diff (expected vs. received):   map[string]string{
      -         "feature.node.kubernetes.io/e2e-attribute-test-1": "true",
      -         "feature.node.kubernetes.io/e2e-flag-test-1":      "true",
      -         "feature.node.kubernetes.io/e2e-instance-test-1":  "true",
        }
      
  occurred
  In [It] at: /go/code/github.com/k8stopologyawareschedwg/node-feature-discovery/test/e2e/node_feature_discovery.go:472
```
2.
```
ESC[1mSTEP:ESC[0m Getting target node and checking labels ESC[38;5;243m11/21/22 12:00:43.205ESC[0m
Nov 21 12:00:43.207: FAIL: label not found!
Expected
    <bool>: false
to be true

Full Stack Trace
sigs.k8s.io/node-feature-discovery/test/e2e.glob..func1.1.5.1()
/go/code/github.com/k8stopologyawareschedwg/node-feature-discovery/test/e2e/node_feature_discovery.go:418 +0x1370
```

**What you expected to happen**:
It should passes

**How to reproduce it (as minimally and precisely as possible)**:
Run the tests on kind cluster:
`make  e2e-test`

**Anything else we need to know?**:
N/A

**Environment**:
- Kubernetes version: 
```
Client Version: v1.25.0
Kustomize Version: v4.5.7
Server Version: v1.25.0
```
- Others: kind cluster

**Comment by marquiz** [comment]
Fixed by #1004
ping @Tal-or  
/close

**Comment by Tal-or** [comment]
Thank you @marquiz
